### ISSUE TITLE
MiHoYo instance on this

### ISSUE THREAD

**Issue by VerifiedGruber** [dovezp/genshin-impact.blacklist#1]
I added this hosts list to my local DNS server blacklist, but as someone who has spent almost a thousand dollars in this game, I can't help but wonder if that's a bannnable offense?

**Comment by dovezp** [comment]
As far as I know, there is nothing contradicting the EULA / TOS from a violation for blocking certain domains that may or may not carry fingerprinting information. As for the third party domains, including Unity, there should be no issues disrupting MiHoYo itself from running. 

So far I only noticed attempts to connect with these services when logging in, bans, and crashes. The game itself doesn't run off of these services luckily, so I've yet to see much impact to any gameplay. I have yet to fully examine the internal packets of the game so there may be some packets being sent if a domain is unable to be reached as the game itself spits out lots of debug logs when it is unable to connect to several hard-coded servers and throws quite the tantrum.  

If you want to play it safe, a proxy server returning 200's and 'spoofed' responses seems to be the best countermeasure as it is pretty much undetectable to any hard-coded client checks within the game than it is making the domains unresponsive. I may or may not include some "fancy hacks" like this in my future bypass release.

Long story short it would be up to MiHoYo to determine if it is considered an offense.
